dom0/sys-usb.sls

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# Installs udev configuration in sys-usb for automatically attaching
+# USB devices to sd-export-usb.
+##
+
+sd-udev-rules:
+  file.managed:
+    - name: /etc/udev/rules.d/99-sd-export-usb.rules
```

Comment:
These rules will not persist accross reboots, since sys-usb is an AppVM. I just did a quick investigation and it appears `/usr/lib/udev/rules.d/` also does not persist reboots. We should investigate how qubes persists their custom udev rules, or perhaps add a script in /rw/ to ensure persistence.

Reply:
Now copying the rules under `/rw/config` and installing them via `/rw/config/rc.local`.